app/components/download-graph.js

```diff
@@ -5,6 +5,7 @@ import Component from '@glimmer/component';
 
 import subDays from 'date-fns/subDays';
 import window from 'ember-window-mock';
+import semverSort from 'semver/functions/sort';
```

Comment:
if we import it here we should explicitly add it as a dependency to the `package.json` file :)

Reply:
You are right, I didn't notice that this is included only in devDependencies. Can I just copy and paste it to dependencies?

Reply:
yeah, just move it to the `dependencies` section